docs/Using-Fleet/MDM-macOS-setup.md

```diff
@@ -0,0 +1,75 @@
+# macOS setup
+
+In Fleet you can customize the first time macOS setup experience for your end users.
+
+You can require end users to authenticate with your identity provider (IdP) and agree to and end user license agreement (EULA) before they can use their new Mac.
```

Comment:
I would recommend turning this into a bulleted list. Those are more scannable and signals to the reader to pay attention. One additional clue to use bulleted lists here is the repeated similar construction: you can... you can...

Reply:
Great point. I turned this into a list